doc/sphinx/source/recipes/recipe_oceans.rst

```diff
@@ -199,16 +199,24 @@ model2 minus obs    model1 minus obs
 
 These figures are also known as Model vs Model vs Obs plots.
 
+The figures produced by this recipe are intended to replicate Marine Assess plots.
+Plots produced are: 
+  * Sea Surface Temperature
```

Comment:
The documentation build is failing, I believe due to the fact that a new line is needed here:
```suggestion

  * Sea Surface Temperature
```